### PR TITLE
Add SyncBatchEmitter

### PR DESCRIPTION
**Pull request by asoltysik**

### CODE REVIEW

**Comment by asoltysik** [comment]
This is now rebased on top the second PR, so can be reviewed independently.

File: modules/id-emitter/src/main/scala/com/snowplowanalytics/snowplow/scalatracker/emitters/id/SyncBatchEmitter.scala [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package com.snowplowanalytics.snowplow.scalatracker.emitters.id
```
Comment (chuwy) [inline modules/id-emitter/src/main/scala/com/snowplowanalytics/snowplow/scalatracker/emitters/id/SyncBatchEmitter.scala:14]: Header

**Review by chuwy — CHANGES_REQUESTED** [review]
LGTM. Just add a header please.

**Review by BenFradet — APPROVED** [review]
:'(

**Pull request merged**